### DISCUSSION TITLE
How to put uptime and viewer count at top of Chatterino?

### DISCUSSION THREAD

**Discussion by WhereSetting** [Chatterino/chatterino2#5740] [Q&A] [answered]
I see other people have it so it shows the uptime and viewer count at the top of Chatterino next to the channel name, I am on 2.5.1 and the default setting only shows it when I put the mouse over the channel name.

**Comment by Felanbird** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/f607f773-e5f5-4ff1-8cd4-69bd68faa660)
